apps/console/src/extensions/components/logs/components/infinite-scroll-container.tsx

```diff
@@ -303,19 +453,43 @@ const InfiniteScrollContainer = (props: InfiniteScrollContainerPropsInterface):
                                                 }
                                             </div>)
                                         }
-                                        { (logType === TabIndex.AUDIT_LOGS) &&
-                                            (<div>
-                                                { log.initiatorId }
-                                            </div>)
-                                        }
-                                        { log.actionId && (
-                                            <div className="log-actionid-container">
-                                                { log.actionId }
+                                        { (logType === TabIndex.AUDIT_LOGS) && (
+                                            <div>
+                                                { log.actionId && (
+                                                    <div className="log-actionid-container">
+                                                        { log.actionId }
+                                                    </div>
+                                                ) }
+                                                { log.action && (
+                                                    <div className="log-actionid-container">
+                                                        { log.action }
+                                                    </div>
+                                                ) }
+                                            </div>
+                                        ) }
+                                        { logType === TabIndex.DIAGNOSTIC_LOGS && (
+                                            <div>
+                                                { log.actionId && (
+                                                    <div className="log-actionid-container">
+                                                        { log.actionId }
+                                                    </div>
+                                                ) }
+                                                { log.action && (
+                                                    <div className="log-actionid-container">
+                                                        { log.action }
+                                                    </div>
+                                                ) }
```

Comment:
Seems like the same jsx component is duplicated here. Will we be able to use only one?

Reply:
Conditionally rendered. This has to be improved.